app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @user = ApplicationController.new.current_user
+    add_like = Like.create(author: @user, post: @post)
```

Comment:
- [ ] the previous comments apply to this part of the code as well, which might look something like `@like = @current_user.likes.new(post_id: @post.id)`